packaging/debian/control.base

```diff
@@ -5,7 +5,7 @@ Priority: optional
 Architecture: all
 Essential: no
 Maintainer: github.com/Shopify/shopify-app-cli
-Depends: ruby (>= 2.3), git (>= 2.13)
+Depends: ruby (>= 2.3), git (>= 2.13), sudo
```

Comment:
And adding `sudo` for Debian doesn't create the same problem as for Fedora-based systems?

Reply:
It does not, I tested removing it and it worked, `sudo` gets removed alongside `shopify-cli`.